lib/ansible/modules/network/netconf/netconf_config.py

```diff
@@ -270,6 +284,9 @@ def main():
             retkwargs=retkwargs,
             datastore=datastore,
         )
+        if changed and module.params['save']:
```

Comment:
For `startup` datastore there is a check in your code to identify if server supports `startup` capability or not and `running` datastore is supported by all Netconf server's, so I am not clear on the limitation of getting diff between `startup` and `running` config.